.github/ISSUE_TEMPLATE/intent-to-implement--i2i-.md

```diff
@@ -48,8 +56,20 @@ The launch tracker is meant to be an easy way of sharing a project's status with
 You should add a link to the launch tracker for this work here if applicable. One ideal template and instructions can be found here: bit.ly/amp-launch-tracker
 -->
 
+## Consideration checklist
+### Design
+- [ ] Design shows understanding of problem statement/priority
+- [ ] Acceptance/success criteria for problem being solved is clear
+- [ ] Design socialized
```

Comment:
Reading the link provided below, it's not clear what "Design socialized" means here.  Could it be "Brought to design review"? Or is there a different intention?

Reply:
Yup it specifically says that later so just changed "Design socialized" to "Brought to Design Review"